app/components/StatusBadge.tsx

```diff
@@ -31,17 +31,13 @@ const colours = {
     badgeColour: "#E6EBF0",
     label: "In review",
   },
-  customText: {
-    labelColour: "#666666",
-    badgeColour: "#E8E8E8",
-  },
 };
 
-const StatusBadge: React.FC<Props> = ({ label = "custom text", variant }) => {
+const StatusBadge: React.FC<Props> = ({ label, variant }) => {
   return (
     <>
       <Chip
-        label={variant === "customText" ? label : colours[variant].label}
+        label={label || colours[variant].label}
```

Comment:
```suggestion
        label={label ?? colours[variant].label}
```

using the nullish coalescing would be more accurate here (same behaviour)